tests/python/test_python_frontend.py

```diff
@@ -4577,13 +4358,9 @@ def fusion_func(
                 dropout_p = 0.0 if dropout_p is None else dropout_p
                 is_causal = False if is_causal is None else is_causal
 
-                with sdpa_kernel(SDPBackend.FLASH_ATTENTION):
-                    torch.manual_seed(0)
-                    ref_out = F.scaled_dot_product_attention(
-                        *qkv, dropout_p=dropout_p, is_causal=is_causal, scale=scale
-                    )
+                with torch.random.fork_rng(devices=[torch.cuda.current_device()]):
+                    with sdpa_kernel(SDPBackend.FLASH_ATTENTION):
+                        ref_out = F.scaled_dot_product_attention(
+                            *qkv, dropout_p=dropout_p, is_causal=is_causal, scale=scale
+                        )
                 torch.testing.assert_close(nvf_out[0], ref_out)
-
-
-if __name__ == "__main__":
-    run_tests()
```

Comment:
Invoking tests through `pytest` does not need this.
@xwang233, in the CI do we invoke tests using `python` anywhere instead of `pytest`?

Reply:
we're using `python` now, but we can switch to `pytest` if that fits better?

Reply:
Oh interesting. What about `pytest_ops.py` ?
@kevinstephano @rdspring1 Do you have any concerns about switching to `pytest.`

Reply:
`pytest` is actually `python -m pytest`, so you can do
```
(gdb) r -m pytest ....py
```